src/utils/database-common/src/transactions/db_transaction_manager.rs

```diff
@@ -102,8 +127,9 @@ impl TransactionRef {
     pub fn into_maybe_transaction<DB: sqlx::Database>(
         self,
     ) -> Option<sqlx::Transaction<'static, DB>> {
-        let m = Arc::try_unwrap(self.inner).unwrap().into_inner();
```

Comment:
```rust
let m = Arc::try_unwrap(self.inner).unwrap().into_inner();
//                                  ^^^^^^^^
``` 

With a missing connection, was getting a crash here -- updated to see a readable error in cli logs